### PR TITLE
Release v1.12.2

### PR DESCRIPTION
**Pull request by lmakarov**
- Fixes
  - Allowed custom commands and addons to be located in symlinked directories (#1036)
  - Added a missing `--rm` in a `docker run` call in `is_docker_path` to cleanup existed containers (#1039, #1064)
  - Refactored `fin cleanup`
    - Not cleaning up dangling volumes not created/managed under Docksal (#1056)
  - Improve error reporting:
    - Error out if `DOCKSAL_STACK` definition is incorrect (#1051)
    - Error out `fin exec` if target container is not found (#1051)
  - Removed `UNISON_USER` and `UNISON_GROUP` from `volumes-unison.yml` (#1049)
    - They are not needed anymore due to docksal/unison#7
  - Reverted to using `wmic.exe` to get logical drives list on Windows (#1059)
  - Added a trigger to short `fin pull` tests in CI
- Documentation
  - Added docs on available Docksal Stock Images https://docs.docksal.io/stack/images-versions/
  - Updated docs on the `DOCKSAL_ENVIRONMENT` variable use cases (#1041)
  - Updated docs for web service settings and multiple webs use case
    - Added a more meaningful example for additional virtual hosts with Apache
    - Added Nginx docs
  - Updated docs on NodeJS use case in `cli`
  - Updated Windows SMB Troubleshooting docs with a case when drives are not shared in Docker Desktop (#1040)
  - Update `fin config` docs (#1035)

### CODE REVIEW

File: docs/content/service/web/settings.md [hunk 1]
```diff
@@ -16,33 +18,71 @@ Use `.docksal/etc/apache/httpd-vhosts.conf` to define additional virtual hosts:
...
+</Directory>
 ```
 
-## Using Different Apache Versions {#apache-versions}
+For a complete list of available features and settings see [docksal/service-apache](https://github.com/docksal/service-apache).
+
+
+### Nginx Advanced Settings {#nginx-settings}
+
+The following settings files in the project codebase can be used to override default Apache settings.
```
Comment (shelane) [inline docs/content/service/web/settings.md]: Should this be Nginx settings, not Apache?

File: docs/content/use-cases/multiple-web.md [hunk 2]
```diff
@@ -1,19 +1,65 @@
...
-backend with a node-based frontend), add a second web service to your `docksal.yml` file:
+For projects that need multiple web servers but need to stay in the same Docksal project (e.g., headless/decoupled 
+applications with frontend and backend hosted separately), you have a few options available. 
 
+
+## Extra Virtual Host
+
+Use the primary web container and add a separate virtual host. See web container [configuration overrides](/service/web/settings/).
+
+All subdomains of you project (`*.project.docksal`) are automatically routed to the primary `web` container.
```
Comment (shelane) [inline docs/content/use-cases/multiple-web.md]: All sub domains of your

File: docs/content/use-cases/nodejs.md [hunk 3]
```diff
@@ -5,8 +5,8 @@ weight: 1
 
 ## Installing Alternate NodeJS Versions
 
-The Docksal `cli` comes pre-installed with NodeJS 8.11.3. This is the recommended version for most users. If you require
-a newer version, you have options:
+The Docksal `cli` comes pre-installed with NodeJS 8.11.3. This is the recommended version for most users.
```
Comment (shelane) [inline docs/content/use-cases/nodejs.md]: 10.16 is now the recommended version. Weren’t we talking about updating?

Comment (shelane) [reply docs/content/use-cases/nodejs.md]: Ah 10.15 is currently installed and we didn’t update the docs.

Comment (lmakarov) [reply docs/content/use-cases/nodejs.md]: Rewrote to say LTS release instead.

File: docs/content/service/web/settings.md (hunk 1 above)
Comment (lmakarov) [reply docs/content/service/web/settings.md]: `service/web/settings.md` now covers both Apache and Nginx. Updated title.

**Pull request merged**